### DISCUSSION TITLE
EmulatorJS 4.0 beta

### DISCUSSION THREAD

**Discussion by ethanaobrien** [EmulatorJS/EmulatorJS#559] [General]
Applications for EmulatorJS 4.0 have now opened! The form will remain open until the beta is ready. The beta will be ready to test soon!

Application form: https://forms.gle/tkHreKDDRcyrEakN9

I am choosing 2 people for the initial beta release

**Comment by Roycek93** [comment]
Touching on the screens a lil unresponsive great overall job illl use it further in custom ROMs for each system congrats
